### DISCUSSION TITLE
Tiny MCE plugins and customization

### DISCUSSION THREAD

**Discussion by HeadStarterz** [AntlerVC/firetable#987] [Support / Q&A] [answered]
Is there a possibility to install more Tiny MCE plugins or premium plugins for the rich text field? Also it would be helpful if I could customize the edit bar myself (e.g. add superscripts and subscripts).

**Comment by harinij** [comment, marked as answer]
@HeadStarterz That would be great, we welcome contributions via PR, is that sometime you would be keen on? You can checkout [this](https://github.com/rowyio/rowy/blob/main/src/components/RichTextEditor.tsx) to get started, it was recently expanded to add full-screen view.
